src/App.js

```diff
@@ -1,15 +1,16 @@
-import React, { Component } from 'react';
-import DataTable from './DataTable';
+import React, { useState, useEffect } from 'react';
+import DataTable from './containers/DataTable';
 import './App.css';
 
-class App extends Component {
-  render() {
-    return (
-      <div className="container mt-3">
-        <DataTable rows={this.props.rows} locale="da" rowsPerPage={5} />
-      </div>
-    );
-  }
-}
+const App = () => {
+  const [userData, setUserData] = useState([]);
+
+  useEffect(() => {
+    const data = JSON.parse(document.getElementById('user-data').dataset.users);
```

Comment:
Data usually come from API, and we usually do the fetching in an effect like so.